src/Mvc/Mvc.ViewFeatures/src/RenderMode.cs

```diff
@@ -30,5 +30,16 @@ public enum RenderMode
         /// </summary>
         ServerPrerendered = 3,
 
+        /// <summary>
+        /// Renders a marker for a Blazor client-side application. This doesn't include any output from the component.
+        /// When the user-agent starts, it uses this marker to bootstrap a blazor client-side application.
+        /// </summary>
+        Client = 4,
```

Comment:
This should be `WebAssembly`. We do not use Client elsewhere to reference to a WASM app.

Reply:
Done

Reply:
That probably means we wanna up the client-side types to `WebAssembly` as well...